pkg/policy/policy_package.go

```diff
@@ -61,14 +63,15 @@ type paramValidatePackage struct {
 	PolicyConfig *ConfigYAML
 }
 
-func (pc *Checker) validatePackage(param *paramValidatePackage) error {
+func (pc *Checker) validatePackage(logE *logrus.Entry, param *paramValidatePackage) error {
 	if param.PolicyConfig == nil {
 		return nil
 	}
 	for _, policyPkg := range param.PolicyConfig.Packages {
 		f, err := pc.matchPkg(param.Pkg, policyPkg)
 		if err != nil {
-			return err
+			logerr.WithError(logE, err).Debug("check if the package matches with a policy")
```

Comment:
If it fails to check if the policy matches with the package, outputs debug log and treats as the policy doesn't match with the package.